lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -150,7 +150,7 @@ def self.run(meth, *args)
       cmd = Specinfra.command.get(meth, *args)
       backend = Specinfra.backend
       ret = backend.run_command(cmd)
-      if meth.to_s =~ /^check/
+      if /^check/.match?(meth.to_s)
```

Comment:
Any thoughts on:
```suggestion
      if meth.to_s.start_with('check')

Reply:
ah I think that's even a tiny bit faster.